hoomd/md/pytest/test_methods.py

```diff
@@ -7,6 +7,7 @@
 _method_data = [
     ("Langevin", hoomd.md.methods.Langevin(hoomd.filter.All(), kT=1, seed=1)),
     ("NVT", hoomd.md.methods.NVT(hoomd.filter.All(), kT=1, tau=1)),
+    ("NPT", hoomd.md.methods.NPT(hoomd.filter.All(), kT=1, tau=1, S=1, tauS=1, box_dof=[True,True,True,False,False,False])),
 ]
```

Comment:
Let's add some NPT specific tests that validate all the parameters that can be correctly set and query both before and after attaching.